tb_plugin/torch_tb_profiler/profiler/loader.py

```diff
@@ -51,11 +52,27 @@ def _parse(self):
                                self.run.name, worker, ex, exc_info=True)
 
     def _process(self):
+        comm_node_lists = []
         for data in self.run.profiles.values():
             logger.debug("Processing profile data")
             data.process()
+            comm_node_lists.append(data.comm_node_list)
             logger.debug("Processing profile data finish")
 
+        worker_num = len(comm_node_lists)
+        for i in range(len(comm_node_lists[0])):
```

Comment:
use enumerate?

Reply:
the index is needed in the loop , this can not use enumerate.

Reply:
thx, done